startup/gui/bookmarks.py

```diff
@@ -45,3 +45,15 @@
 
 fontBookmarks = GafferUI.Bookmarks.acquire( application, category="font" )
 fontBookmarks.add( "Gaffer Fonts", os.path.expandvars( "$GAFFER_ROOT/fonts" ) )
+
+shaderBookmarks = GafferUI.Bookmarks.acquire( application, category="shader" )
+defaultShaderDirectory = os.path.expandvars( "$HOME/gaffer/shaders" )
+try :
```

Comment:
That form is vulnerable to race conditions though - the UI startup code maybe doesn't have a significant probability of hitting it (you'd need to start two GUI sessions in quick succession), but as a general rule it's something to avoid.